internal/sql/repository/chartConfig/ConfigMapRepository.go

```diff
@@ -58,6 +60,39 @@ type ConfigMapAppModel struct {
 	SecretData    string   `sql:"secret_data"`
 	sql.AuditLog
 }
+type CMCSNames struct {
+	CMName string `json:"cm_name"`
+	CSName string `json:"cs_name"`
+}
+
+func (impl ConfigMapRepositoryImpl) GetConfigNamesAppLevel(appId int) ([]CMCSNames, error) {
+	var cMCSNames []CMCSNames
+	query := "SELECT json_array_elements(config_map_data::json->'maps')->>'name' AS cm_name,    " +
+		"json_array_elements(secret_data::json->'secrets')->>'name' AS cs_name " +
+		"FROM config_map_app_level WHERE app_id = ?;"
```

Comment:
why raw query, use go-pg for where clauses etc and can put your expression in select.
I think we had discussed this already